docs/source/en/training/lora.md

```diff
@@ -120,7 +120,6 @@ Load the LoRA weights from your finetuned model *on top of the base model weight
 ... ).images[0]
 # use the weights from the fully finetuned LoRA model
 
->>> image = pipe("A pokemon with blue eyes.", num_inference_steps=25, guidance_scale=7.5).images[0]
```

Comment:
I believe this is intentional, the first pipeline demonstrates `cross_attention_kwargs` to only use some of the LoRA weights, and the second pipeline shows using the full LoRA weights. Maybe we can just comment out the second example `pipe`?

Reply:
Oh OK, that part wasn't clear to me.  So, when `cross_attention_kwargs` is omitted (and in turn, no `scale` parameter provided), then it is the same as having `scale` set to `1`?

Reply:
I've inserted the second pipeline back into the example, but as a comment, with clarification that you use either one OR the other.  Also moved the comment above nearest to the code where it is most relevant.

Reply:
Yup! 👍